kalite/main/content_rating_models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+from kalite.facility.models import FacilityUser
+
+from securesync.models import DeferredCountSyncedModel
+
+class ContentRating(DeferredCountSyncedModel):
+    class Meta:
+        unique_together = ("content_kind", "content_id", "user")
+
+    # Maintain info on content type -- could be video, exercise, etc, but we should be able to uniquely id it
+    content_kind = models.CharField(max_length=100, db_index=True, blank=False)
+    content_id = models.CharField(max_length=100, db_index=True, blank=False)
+

```

Comment:
Need to add "content_source" in here as well, so we'll later be able to distinguish channels more easily. See https://github.com/MCGallaspy/ka-lite/blob/beautify-content-rating/kalite/main/models.py#L531